### DISCUSSION TITLE
won't install

### DISCUSSION THREAD

**Discussion by ProgramFilesx86** [projectdiscovery/shuffledns#78] [Q&A] [answered]
Hello everyone,

I just followed the installation process , however the tool didn't installed,

I ran this command on kali linux 2021.2 
$ GO111MODULE=on go get -v github.com/projectdiscovery/shuffledns/cmd/shuffledns

it did downloaded as well, but when I do "shuffledns" "shuffledns.go" ...etc it returns "command not found"

I also tried to clone the repository>go get>go build 
and yet the script still not installed

Please can anyone improve the README.md about the installation and thanks in advance

**Comment by forgedhallpass** [comment, marked as answer]
`go get` downloads packages under `GOPATH`, or if the environmental variable is not set I believe it defaults to `$HOME/go`, hence you should be looking there. (e.g. `$GOPATH/bin/shuffledns` or `~/go/bin/shuffledns`)

For the cloned repository:
```shell
cd shuffledns/cmd/shuffledns
go build -o shuffledns
chmod +x shuffledns
./shuffledns 
```

**Reply by 0xCGonzalo** [reply]
Thanks!

**Comment by ehsandeep** [comment]
@ProgramFilesx86 `go install -v github.com/projectdiscovery/shuffledns/cmd/shuffledns@latest` additionally you can do this as well.
